### ISSUE TITLE
Return back to the home screen after closing an app

### ISSUE THREAD

**Issue by sandorfimarci** [MM2-0/Kvaesitso#189]
When the user open an app from a search and press the home button, the result of the search will be shown not the homescreen (the one with the clock).

**Reproduce:**
1. Search for e.g. an app.
2. Open that app from the result list.
3. After the app is opened press the home button.

**Requested behavior:** 
See the homescreen, with the clock.

**Current behavior:** 
The result of the search.

**Comment by mitnworb** [comment]
I would like to see this too.  Without it, it adds an extra swipe to get back to the home screen.

**Comment by MM2-0** [comment]
This has already been implemented (in 1.22). After a timeout of 60 seconds, the launcher will clear the search bar and return to home screen when you return to the launcher (from an app or from lock screen).

60 seconds timeout to account for misclicks or when you are actually searching for something and want to launch multiple search results for the same query.

**Comment by mitnworb** [comment]
Great.  How about the same thing for when I launch an app from the Favorites Widget?  I can open a new feature request is you like.  Let me know.

Reproduce:

    Scroll down to Favorites Widget.
    Open that app from the widget.
    After the app is opened press the home button.

Requested behavior:
See the homescreen, with the clock.

Current behavior:
Show the widgets.

**Comment by MM2-0** [comment]
Good point, I can add that as well

**Comment by mitnworb** [comment]
What about a settings option to allow the end user to set the timeout lower than 60 sec?  Sometimes I jump into my mail app for 10-20 seconds and then back to the homescreen.

**Comment by dongltron** [comment]
Agreed, I would appreciate this. I often hop between social apps and if I have no messages I will quit. Usually takes like 20 seconds, but the option to reset any time or as suggested above would be nice (including 0 seconds). If you really wanna stick with the 'accidental tap' route though, surely 5 seconds is enough. Somebody isn't gonna take 60 seconds to register they tapped the wrong thing!
